### ISSUE TITLE
Share teacher and student pretrained model?

### ISSUE THREAD

**Issue by Cv9527** [cuguilke/microexpnet#2]
@cuguilke Thanks for sharing the code of microexpnet,  could you share the pretrained models of teacher and student for testing?

**Comment by cuguilke** [comment]
Student models are available now.

**Comment by cuguilke** [comment]
teacher model for Oulu-CASIA is available, CK+ model is on the way...

**Comment by cuguilke** [comment]
done

**Comment by speculaas** [comment]
Dear Cugu, 
Thanks for your help!

Can you share how to do distillation?

I briefly read through your code, I see the following line: 
./Candidates/trainCandidates.py:95:                     testX, testY, testSoftTargets = deployImages(valSet, teacherSoftmaxInputs)

But I cannot find how to put teacher model's logits into the teacherSoftmaxInputs variable. 
BR,
JimmyYS

**Comment by cuguilke** [comment]
You don't. Candidates are trained without distillation. I updated the code, there shouldn't be a variable called "teacherSoftmaxInputs" at the first place.
